core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/Integration.kt

```diff
@@ -156,6 +137,7 @@ internal class Integration(
         import("org.jetbrains.kotlinx.dataframe.columns.*")
         import("org.jetbrains.kotlinx.dataframe.jupyter.ImportDataSchema")
         import("org.jetbrains.kotlinx.dataframe.jupyter.importDataSchema")
+        import("org.jetbrains.kotlinx.dataframe.jupyter.getRowsSubsetForRendering")
```

Comment:
Why should this be imported by default? You want people to use this as an API or there is another reason?

Reply:
The Swing tables component uses this function to query subsets of the dataframe. This way, pagination is implemented. It can be imported on every query, but It makes sense to import once because it's a frequent operation.

Reply:
Hm, i don't fully understand how this integration works. Where is this function called? I don't see usages in dataframe repository, and at the same time i don't understand how it could be possible to call dataframe function from the plugin

Reply:
I mean, it looks like this function will be called from generated code, but i somehow missed where this code is generated

Reply:
In this case, the IDEA plugin is just a web client of the Jupyter kernel. The plugin sends Kotlin code to the kernel to evaluate and asynchronously receives a response.
something like this 
``
DISPLAY(getRowsSubsetForRendering(Out[x], 0, 20), "")
``

Reply:
Ok, i understand now. Could you add this explanation as a comment?  Also, I've been thinking about limiting the scope of this function somehow. Apparently we need it as a part of public API, but now it's also a global function. Would be good to make it not global